src/components/sideMenu/index.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import { Drawer, Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+function SideMenu() {
+  const drawerWidth = 300;
+  const useStyles = makeStyles((theme) => ({
+    drawer: {
+      width: drawerWidth,
+      flexShrink: 0,
+    },
+    drawerPaper: {
+      width: drawerWidth,
+      backgroundColor: '#F0F0F0',
+    },
+
+    logo: {
+      textAlign: 'center',
+      marginTop: '30px',
+    },
+    logoNisum: {
+      color: '#e91e63',
```

Comment:
Do we need to use hard coded colour still after defining our theme .. If it is a colour that isn't in the theme then try to add a constant to the Color file and use that constant here !!